src/app/public/cell-editors/cell-editor-datepicker/cell-editor-datepicker.component.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  ViewChild
+} from '@angular/core';
+
+import {
+  ICellEditorAngularComp
+} from 'ag-grid-angular';
+
+import {
+  SkyDatepickerInputDirective
+} from '@skyux/datetime';
+
+import {
+  ICellEditorParams
+} from 'ag-grid-community';
+
+@Component({
+  selector: 'sky-cell-editor-datepicker',
+  templateUrl: './cell-editor-datepicker.component.html',
+  styleUrls: ['./cell-editor-datepicker.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyCellEditorDatepickerComponent implements ICellEditorAngularComp {
+  private params: ICellEditorParams;
+
+  @ViewChild('skyCellEditorDatepickerInput', {read: ElementRef})
+  public datepickerInput: ElementRef;
+
+  @ViewChild(SkyDatepickerInputDirective)
+  public inputDirective: SkyDatepickerInputDirective;
+
+  public currentDate: Date;
+  public minDate: Date;
+  public maxDate: Date;
+  public disabled: boolean;
+  public dateFormat: string;
+  public startingDay: number;
+  public columnWidth: number;
+  public rowHeight: number;
+
+  constructor(private el: ElementRef) {}
+
+  public agInit(params: ICellEditorParams) {
+    this.params = params;
+    this.currentDate = this.params.value;
+    const cellEditorParams = this.params.colDef.cellEditorParams;
+    this.minDate = cellEditorParams && cellEditorParams.minDate;
```

Comment:
Wrap this in `if (cellEditorParams) {` and remove redundant `cellEditorParams &&`.